README.md

```diff
@@ -1 +1,60 @@
-# To_do_list
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
+
+> In this project we are making a library where you can save all your favorite books!
+
+## Built With
+
```

Comment:
- [ ] It would be nice if you updated the README file to contain information about the current project.